compiler/src/dotty/tools/dotc/coverage/Serializer.scala

```diff
@@ -0,0 +1,83 @@
+package dotty.tools.dotc
+package coverage
+
+import java.nio.file.{Path, Paths, Files}
+import java.io.Writer
+import scala.language.unsafeNulls
+
+/**
+ * Serializes scoverage data.
+ * @see https://github.com/scoverage/scalac-scoverage-plugin/blob/main/scalac-scoverage-plugin/src/main/scala/scoverage/Serializer.scala
```

Comment:
We should add a note in https://github.com/lampepfl/dotty/blob/main/NOTICE.md listing which packages contain files copied from scoverage

Reply:
Well, it's been modified since the initial copy. But that surely counts as well ^^